app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/viewmode.phtml

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+?>
```

Comment:
Remove Magento copyrights please, and add template annotation about changes

Reply:
Copied folder "toolbar" with all files. File  viewmode deleted.